roles/lib_openshift/src/doc/certificate_authority

```diff
@@ -0,0 +1,141 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oadm_ca
+short_description: Module to manage openshift certificate authority
+description:
+  - Wrapper around the openshift `oc adm ca` command.
+options:
+  state:
```

Comment:
ack, makes sense. I misread the intention of the exists method in skimming the PR.